R/shiny_prerendered.R

```diff
@@ -159,7 +160,14 @@ shiny_prerendered_html <- function(input_rmd, render_args) {
     dependencies <- append(dependencies, list(html_dependency_rsiframe()))
 
   # return html w/ dependencies
-  shinyHTML_with_deps(rendered_html, dependencies)
+  html_with_deps <- shinyHTML_with_deps(rendered_html, dependencies)
+
+  # add placeholder for additional dependencies at the end of the <head> element
+  sub('</head>',
+      '\n<!-- HEAD_CONTENT -->\n</head>',
```

Comment:
```suggestion
      '\n{{ headContent() }}\n</head>',      
```

Reply:
Given my suggestion above (to not send this through `htmlTemplate`) this suggestion would no longer work, but something like this would:
```suggestion
      paste0('\n', htmlTemplate(text_ = "{{ headContent() }}"), '\n</head>'),
```